src/System.Security.Cryptography.Pkcs/tests/SignedCms/SignerInfoTests.netcoreapp.cs

```diff
@@ -178,6 +180,47 @@ public static void SignerInfo_RemoveUnsignedAttributes_MultipleAttributeValues()
             Assert.Equal(0, cms.SignerInfos[0].UnsignedAttributes.Count);
         }
 
+        [Fact]
+        public static void SignerInfo_AddRemoveUnsignedAttributes_JoinCounterSignaturesAttributesIntoOne()
```

Comment:
this test can be used to create a repro document for the RemoveCounterSignature issue